worlds/sm64ex/__init__.py

```diff
@@ -35,22 +35,53 @@ class SM64World(World):
     item_name_to_id = item_table
     location_name_to_id = location_table
 
-    data_version = 8
+    data_version = 9
     required_client_version = (0, 3, 5)
 
     area_connections: typing.Dict[int, int]
 
     option_definitions = sm64_options
 
+    number_of_stars: int
+    move_rando_bitvec: int
+    filler_count: int
+    star_costs: typing.Dict[str, int]
+
     def generate_early(self):
+        max_stars = 120
+        if (not self.multiworld.EnableCoinStars[self.player].value):
+            max_stars -= 15
+        self.move_rando_bitvec = 0
+        for action, itemid in action_item_table.items():
+            # HACK: Disable randomization of double jump
+            if action == 'Double Jump': continue
+            if getattr(self.multiworld, f"MoveRandomizer{action.replace(' ','')}")[self.player].value:
+                max_stars -= 1
+                self.multiworld.itempool.append(self.create_item(action))
```

Comment:
Can you move this to `create_items`? Its technically allowed here, but it would be more consistent with the other items.

Reply:
Missed this comment.  Moved item creation to `create_items` using the same conventions as checking for items in rules.